Shared class cache embedded in the container is not used

I experimented with latest versions of `openliberty/open-liberty:20.0.0.12-kernel-slim-java8-openj9-ubi` and `openliberty/open-liberty:20.0.0.12-full-java8-openj9-ubi` and saw an increased startup time.
Upon further inspection it turns out that Liberty is trying to use a shared class cache (SCC) from `/opt/ol/wlp/output/.classCache` directory which does not exist (it will be created and populated at runtime, but this means increased start-up time). In the `configure.sh` script Liberty correctly determines that the docker layer it is based on, `adoptopenjdk/openjdk8-openj9:ubi`, has an embedded SCC at `/opt/java/.scc` and it builds another SCC layer on top it. However, at runtime, it decides to stick with the old `/opt/ol/wlp/output/.classCache` location.
I think this is an artifact of a recent change in the `adoptopenjdk/openjdk8-openj9:ubi` image where `OPEN9_JAVA_OPTIONS` is no longer defined. This change allows an user to provide their own JDK options through the documented `OPENJ9_JAVA_OPTIONS` environment variable, but it seems to have broken Liberty container behavior.

javacore shows the following arguments:
```
2CIUSERARG               -Xshareclasses:name=openj9_system_scc,cacheDir=/opt/java/.scc,readonly,nonFatal
2CIUSERARG               -Xshareclasses:name=liberty-%u,nonfatal,cacheDir=/opt/ol/wlp/output/.classCache,cacheDirPerm=1000
2CIUSERARG               -XX:ShareClassesEnableBCI
2CIUSERARG               -Xscmx80m
2CIUSERARG               -Xshareclasses:name=liberty-%u,nonfatal,cacheDir=/opt/ol/wlp/output/.classCache,cacheDirPerm=1000
2CIUSERARG               -XX:ShareClassesEnableBCI
2CIUSERARG               -Xscmx80m
```
Note the multiple definition of the -Xshareclasses command line option

@ymanton @jdmcclur

--- a/releases/latest/beta/Dockerfile.ubuntu.adoptopenjdk11
+++ b/releases/latest/beta/Dockerfile.ubuntu.adoptopenjdk11
@@ -87,7 +87,8 @@ RUN if [ "$OPENJ9_SCC" = "true" ]; then populate_scc.sh; fi \
     && chmod -R g+rwx /opt/ol/wlp/output
 
 #These settings are needed so that we can run as a different user than 1001 after server warmup
-ENV RANDFILE=/tmp/.rnd
+ENV RANDFILE=/tmp/.rnd \
+    OPENJ9_JAVA_OPTIONS="-XX:+IgnoreUnrecognizedVMOptions -XX:+IdleTuningGcOnIdle -Xshareclasses:name=openj9_system_scc,cacheDir=/opt/java/.scc,readonly,nonFatal"
 
 USER 1001
 

--- a/releases/latest/beta/Dockerfile.ubuntu.adoptopenjdk8
+++ b/releases/latest/beta/Dockerfile.ubuntu.adoptopenjdk8
@@ -87,7 +87,8 @@ RUN if [ "$OPENJ9_SCC" = "true" ]; then populate_scc.sh; fi \
     && chmod -R g+rwx /opt/ol/wlp/output
 
 #These settings are needed so that we can run as a different user than 1001 after server warmup
-ENV RANDFILE=/tmp/.rnd
+ENV RANDFILE=/tmp/.rnd \
+    OPENJ9_JAVA_OPTIONS="-XX:+IgnoreUnrecognizedVMOptions -XX:+IdleTuningGcOnIdle -Xshareclasses:name=openj9_system_scc,cacheDir=/opt/java/.scc,readonly,nonFatal"
 
 USER 1001
 

--- a/releases/latest/full/Dockerfile.ubi.adoptopenjdk11
+++ b/releases/latest/full/Dockerfile.ubi.adoptopenjdk11
@@ -88,7 +88,8 @@ RUN if [ "$OPENJ9_SCC" = "true" ]; then populate_scc.sh; fi \
     && chmod -R g+rwx /opt/ol/wlp/output
 
 #These settings are needed so that we can run as a different user than 1001 after server warmup
-ENV RANDFILE=/tmp/.rnd
+ENV RANDFILE=/tmp/.rnd \
+    OPENJ9_JAVA_OPTIONS="-XX:+IgnoreUnrecognizedVMOptions -XX:+IdleTuningGcOnIdle -Xshareclasses:name=openj9_system_scc,cacheDir=/opt/java/.scc,readonly,nonFatal"
 
 USER 1001
 

--- a/releases/latest/full/Dockerfile.ubi.adoptopenjdk15
+++ b/releases/latest/full/Dockerfile.ubi.adoptopenjdk15
@@ -88,7 +88,8 @@ RUN if [ "$OPENJ9_SCC" = "true" ]; then populate_scc.sh; fi \
     && chmod -R g+rwx /opt/ol/wlp/output
 
 #These settings are needed so that we can run as a different user than 1001 after server warmup
-ENV RANDFILE=/tmp/.rnd
+ENV RANDFILE=/tmp/.rnd \
+    OPENJ9_JAVA_OPTIONS="-XX:+IgnoreUnrecognizedVMOptions -XX:+IdleTuningGcOnIdle -Xshareclasses:name=openj9_system_scc,cacheDir=/opt/java/.scc,readonly,nonFatal"
 
 USER 1001
 

--- a/releases/latest/full/Dockerfile.ubi.adoptopenjdk8
+++ b/releases/latest/full/Dockerfile.ubi.adoptopenjdk8
@@ -88,7 +88,8 @@ RUN if [ "$OPENJ9_SCC" = "true" ]; then populate_scc.sh; fi \
     && chmod -R g+rwx /opt/ol/wlp/output
 
 #These settings are needed so that we can run as a different user than 1001 after server warmup
-ENV RANDFILE=/tmp/.rnd
+ENV RANDFILE=/tmp/.rnd \
+    OPENJ9_JAVA_OPTIONS="-XX:+IgnoreUnrecognizedVMOptions -XX:+IdleTuningGcOnIdle -Xshareclasses:name=openj9_system_scc,cacheDir=/opt/java/.scc,readonly,nonFatal"
 
 USER 1001
 

--- a/releases/latest/full/Dockerfile.ubuntu.adoptopenjdk11
+++ b/releases/latest/full/Dockerfile.ubuntu.adoptopenjdk11
@@ -87,7 +87,8 @@ RUN if [ "$OPENJ9_SCC" = "true" ]; then populate_scc.sh; fi \
     && chmod -R g+rwx /opt/ol/wlp/output
 
 #These settings are needed so that we can run as a different user than 1001 after server warmup
-ENV RANDFILE=/tmp/.rnd
+ENV RANDFILE=/tmp/.rnd \
+    OPENJ9_JAVA_OPTIONS="-XX:+IgnoreUnrecognizedVMOptions -XX:+IdleTuningGcOnIdle -Xshareclasses:name=openj9_system_scc,cacheDir=/opt/java/.scc,readonly,nonFatal"
 
 USER 1001
 

--- a/releases/latest/full/Dockerfile.ubuntu.adoptopenjdk8
+++ b/releases/latest/full/Dockerfile.ubuntu.adoptopenjdk8
@@ -87,7 +87,8 @@ RUN if [ "$OPENJ9_SCC" = "true" ]; then populate_scc.sh; fi \
     && chmod -R g+rwx /opt/ol/wlp/output
 
 #These settings are needed so that we can run as a different user than 1001 after server warmup
-ENV RANDFILE=/tmp/.rnd
+ENV RANDFILE=/tmp/.rnd \
+    OPENJ9_JAVA_OPTIONS="-XX:+IgnoreUnrecognizedVMOptions -XX:+IdleTuningGcOnIdle -Xshareclasses:name=openj9_system_scc,cacheDir=/opt/java/.scc,readonly,nonFatal"
 
 USER 1001
 

--- a/releases/latest/kernel-slim/Dockerfile.ubi.adoptopenjdk11
+++ b/releases/latest/kernel-slim/Dockerfile.ubi.adoptopenjdk11
@@ -87,7 +87,8 @@ RUN if [ "$OPENJ9_SCC" = "true" ]; then populate_scc.sh; fi \
     && chmod -R g+rwx /opt/ol/wlp/output
 
 #These settings are needed so that we can run as a different user than 1001 after server warmup
-ENV RANDFILE=/tmp/.rnd
+ENV RANDFILE=/tmp/.rnd \
+    OPENJ9_JAVA_OPTIONS="-XX:+IgnoreUnrecognizedVMOptions -XX:+IdleTuningGcOnIdle -Xshareclasses:name=openj9_system_scc,cacheDir=/opt/java/.scc,readonly,nonFatal"
 
 USER 1001
 

--- a/releases/latest/kernel-slim/Dockerfile.ubi.adoptopenjdk15
+++ b/releases/latest/kernel-slim/Dockerfile.ubi.adoptopenjdk15
@@ -87,7 +87,8 @@ RUN if [ "$OPENJ9_SCC" = "true" ]; then populate_scc.sh; fi \
     && chmod -R g+rwx /opt/ol/wlp/output
 
 #These settings are needed so that we can run as a different user than 1001 after server warmup
-ENV RANDFILE=/tmp/.rnd
+ENV RANDFILE=/tmp/.rnd \
+    OPENJ9_JAVA_OPTIONS="-XX:+IgnoreUnrecognizedVMOptions -XX:+IdleTuningGcOnIdle -Xshareclasses:name=openj9_system_scc,cacheDir=/opt/java/.scc,readonly,nonFatal"
 
 USER 1001
 

--- a/releases/latest/kernel-slim/Dockerfile.ubi.adoptopenjdk8
+++ b/releases/latest/kernel-slim/Dockerfile.ubi.adoptopenjdk8
@@ -87,7 +87,8 @@ RUN if [ "$OPENJ9_SCC" = "true" ]; then populate_scc.sh; fi \
     && chmod -R g+rwx /opt/ol/wlp/output
 
 #These settings are needed so that we can run as a different user than 1001 after server warmup
-ENV RANDFILE=/tmp/.rnd
+ENV RANDFILE=/tmp/.rnd \
+    OPENJ9_JAVA_OPTIONS="-XX:+IgnoreUnrecognizedVMOptions -XX:+IdleTuningGcOnIdle -Xshareclasses:name=openj9_system_scc,cacheDir=/opt/java/.scc,readonly,nonFatal"
 
 USER 1001
 

--- a/releases/latest/kernel-slim/Dockerfile.ubuntu.adoptopenjdk11
+++ b/releases/latest/kernel-slim/Dockerfile.ubuntu.adoptopenjdk11
@@ -86,7 +86,8 @@ RUN if [ "$OPENJ9_SCC" = "true" ]; then populate_scc.sh; fi \
     && chmod -R g+rwx /opt/ol/wlp/output
 
 #These settings are needed so that we can run as a different user than 1001 after server warmup
-ENV RANDFILE=/tmp/.rnd
+ENV RANDFILE=/tmp/.rnd \
+    OPENJ9_JAVA_OPTIONS="-XX:+IgnoreUnrecognizedVMOptions -XX:+IdleTuningGcOnIdle -Xshareclasses:name=openj9_system_scc,cacheDir=/opt/java/.scc,readonly,nonFatal"
 
 USER 1001
 

--- a/releases/latest/kernel-slim/Dockerfile.ubuntu.adoptopenjdk8
+++ b/releases/latest/kernel-slim/Dockerfile.ubuntu.adoptopenjdk8
@@ -86,7 +86,8 @@ RUN if [ "$OPENJ9_SCC" = "true" ]; then populate_scc.sh; fi \
     && chmod -R g+rwx /opt/ol/wlp/output
 
 #These settings are needed so that we can run as a different user than 1001 after server warmup
-ENV RANDFILE=/tmp/.rnd
+ENV RANDFILE=/tmp/.rnd \
+    OPENJ9_JAVA_OPTIONS="-XX:+IgnoreUnrecognizedVMOptions -XX:+IdleTuningGcOnIdle -Xshareclasses:name=openj9_system_scc,cacheDir=/opt/java/.scc,readonly,nonFatal"
 
 USER 1001
 

